en-US/firefox/home-master.lang

```diff
@@ -0,0 +1,103 @@
+## NOTE: Demo http://www-demo4.allizom.org/firefox/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/
+
+
+# HTML page title
+;Protect your life online with Firefox and a family of privacy-first products
+Protect your life online with Firefox and a family of privacy-first products
+
+
+# HTML description
+;Firefox is more than a browser. Learn more about Firefox products that handle your data with respect and are built for privacy anywhere you go online.
+Firefox is more than a browser. Learn more about Firefox products that handle your data with respect and are built for privacy anywhere you go online.
+
+
+;The browser is just the beginning
+The browser is just the beginning
+
+
+;Meet our family of products
+Meet our family of products
+
+
+;Get 2,000+ trackers off your trail — including Facebook
+Get 2,000+ trackers off your trail — including Facebook
+
+
+;Know when hackers strike — and stay a step ahead
+Know when hackers strike — and stay a step ahead
+
+
+# CTA link
+;Start getting breach reports
+Start getting breach reports
+
+
+;Keep your passwords safe on every device
+Keep your passwords safe on every device
+
+
+# Lockwise is a product name that shouldn't be translated
+;Learn more about Lockwise
+Learn more about Lockwise
+
+
+# the strong tags around "respect" add a special underline, the underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
```

Comment:
I am not sure dropping the tag works during error check. Usually it looks for the same tags in target language. If one or the whole pair is missing, it can't be pushed. This is how [Practice respect ](https://transvision.mozfr.org/string/?entity=mozilla_org/navigation.lang:594e6d4521df8e272177c62c5f4be47a&repo=mozilla_org)is translated

Reply:
Oh good call... I think we might need to add an alternative version without the tags as an option?

Reply:
The graphic does make the word stand out. Since this is not opening up to more locales, I think we are safe for the key markets. Let's deal with it when we open to more locales. May be changes will be in text and layout too.

Reply:
Oh, since this *is* opening to more locales after all, I think we do need the alternative headline without the tag.

Reply:
Hm... except that won't work in the template... bah.